backend/app/controllers/users.js

```diff
@@ -185,12 +190,8 @@ exports.login = async (req, res) => {
 	}
 
 	try {
-		const USER = await prisma.mec_user.findOne({
-			attributes: ["id", "mec_pwd"],
-			where: prisma.sequelize.where(
-				prisma.sequelize.fn("lower", prisma.sequelize.col("mec_un")),
-				prisma.sequelize.fn("lower", email)
-			),
+		const USER = await prisma.user.findUnique({
+			where: {email: body.email},
 		});
 
 		if (!USER) {
```

Comment:
Si no hay usuario, un error ha ocurrido y deberíamos de avisar. Entiendo pasaria en el Catch, pero sino, este condicional debería de hacer un error 404 de NOT FOUND porque esta buscando uno y no existe.